### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YaphetKG**

### CODE REVIEW

File: helm/answer-coalesce/values.yaml [hunk 1]
```diff
@@ -46,6 +46,8 @@ redis:
     size:
     class:
 
+dataUrl: "https://stars.renci.org/var/answer_coalesce/2021-09-24.rdb"
```
Comment (PhillipsOwen) [inline helm/answer-coalesce/values.yaml:49]: there is actually a better version of this redis at /projects/stars/Data_services/AC/2021-09-25.rdb

the permissions on the /var directory didnt permit me to copy it there.

**Review by PhillipsOwen — APPROVED** [review]
looks good. note the comment on the new AC redis dump file.

File: helm/answer-coalesce/values.yaml (hunk 1 above)
Comment (YaphetKG) [reply helm/answer-coalesce/values.yaml:49]: ok will copy those in and update the url

Comment (PhillipsOwen) [reply helm/answer-coalesce/values.yaml:49]: thanks!

**Pull request closed without merging**